docs/source/usage/workflow-installation.rst

```diff
@@ -73,7 +73,8 @@ the command ``cylc gui``.
 
     # Run workflow
     cylc vip .
-    # View workflow progress
+    # View workflow progress and don't use & at the end to enable
+interactive mode.
```

Comment:
```suggestion
    # View workflow progress and don't use & at the end to enable interactive mode.
```
This needs to all be on the comment line. If it does need to go onto multiple it needs to be indented.